src/librbd/io/Utils.cc

```diff
@@ -6,8 +6,10 @@
 #include "include/buffer.h"
 #include "include/rados/librados.hpp"
 #include "include/neorados/RADOS.hpp"
+#include "librbd/internal.h"
 #include "librbd/io/AioCompletion.h"
 #include "librbd/io/ImageRequest.h"
+#include "librbd/Utils.h"
```

Comment:
Nit: move this up to between lines 9 and 10.

Reply:
May I ask why? Any guideline? I am confused about the alphabet order.

Reply:
Sorry, missed this (wasn't ignoring). Just trying to keep it alphabetical by subdirectory (i.e. imagine it's listed as `librbd/./Utils.h`)